src/styles/main.scss

```diff
@@ -0,0 +1,61 @@
+* {
+  font-family: Arial, sans-serif;
+}
+
+$height-block: 100px;
+$width-block: 100px;
+$amount-days-in-row: 7;
+$num-days: 31;
+
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
```

Comment:
The body styling is correctly set up for centering content, which is good for the calendar layout.